src/os/interpolate.js

```diff
@@ -34,9 +34,10 @@ const SettingsKey = {
 let TransformSet;
 
 /**
+ * Local interpolate config.
  * @type {!Config}
  */
-const interpolateConfig = {
+const interpolateConfig_ = {
```

Comment:
Historically we used underscores for private properties so it would be very clear if that property was being referenced outside the file/class/etc (ie, `obj.someProperty_` would be an immediate red flag). For properties local to a module I don't think the underscore is necessary because the property is truly private unless exported.

Reply:
I went with the underscore to follow some of the style I saw you doing with set functions in other modules. This case was actually causing an exception on interpolation change due to the collision in variable names in the `setConfig` function. I can give it another name, but I think indicating that it is private to this module is useful.

Reply:
Ahh yeah, those could have been named better. We started that pattern because those are the exact same function, one is exported and the other is a reference that can be changed. We have always followed Google's JS style guide though, which explicitly states to not use `@private` or end the name in an underscore.

https://google.github.io/styleguide/jsguide.html#file-goog-module-exports